packages/deployment/Dockerfile

```diff
@@ -1,10 +1,13 @@
 ARG TAG=latest
 ARG REPO=agoric/agoric-sdk
-FROM golang:buster AS go-build
 
-WORKDIR /usr/src/journalbeat
-RUN apt-get update -y && apt-get install -y libsystemd-dev
-RUN go get github.com/mheese/journalbeat
+# FIXME: Journalbeat compilation is currently broken, but non-essential.
```

Comment:
We eventually do want to use journalbeat? That's why the FIXME and the commented rather than deleted code?

Reply:
Ah, just paranoia.  I'll delete the code.